theme.toml

```diff
@@ -54,6 +54,7 @@
 "ui.selection.primary" = { bg = "#540099" }
 "ui.cursor.select" = { bg = "#6F44F0" }
 "ui.cursor.insert" = { bg = "#802F00" }
+"ui.cursor.match" = { fg = "#212121", bg = "#757575" }
```

Comment:
Let's make this `bg = #6c6999` to match the dim color that alacritty produces. No fg.

Reply:
```suggestion
"ui.cursor.match" = { fg = "#212121", bg = "#6C6999" }
```